lib/modules/manager/pip-compile/common.ts

```diff
@@ -210,9 +241,13 @@ const pythonVersionRegex = regEx(
 );
 
 export function extractPythonVersion(
+  commandType: CommandType,
   content: string,
   fileName: string,
 ): string | undefined {
+  if (commandType === 'uv') {
+    return;
+  }
```

Comment:
Please add a code comment here explaining why python version isn't relevant to `uv`